src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -4407,7 +4415,6 @@ unsigned int processContextElement
         /* Successful creation: send potential notifications */
         std::map<std::string, TriggeredSubscription*>  subsToNotify;
         std::vector<std::string>                       attrNames;
-        std::vector<std::string>                       attributes;
```

Comment:
Not related with this PR, but I have realized this variable is not needed.

NTC (informative)